community-videos.md

```diff
@@ -3,4 +3,4 @@ layout: page
 permalink: community/videos.html
 ---
 
-{% include articles.html collection="community_videos" %}
\ No newline at end of file
+{% include articles.html collection="community_videos" sort-order="desc" %}
```

Comment:
thanks for the quick turnaround. However, one of the "videos" in the list is actually the intro to the pywhy series. With this sort, it might go to the last. can you try adding that "video", 00_talk_series.md as a text inside the community-videos.md?

Reply:
I moved that 00_talk_series to 99_talk_series so that it will show up at the top in reverse order.

It would take more time to figure out how to add the header in a correctly formatted way to the beginning of community-videos.md